experimental/dds/tree2/src/feature-libraries/sequence-field/invert.ts

```diff
@@ -121,7 +122,7 @@ function invertMark<TNodeChange>(
 							id: mark.id,
 							cellId: outputId,
 							count: mark.count,
-							detachIdOverride: mark.cellId,
+							detachIdOverride: getInputCellId(mark, revision, revisionMetadata),
```

Comment:
Is this related to the rest of the PR?

Reply:
It was needed when I was trying to avoid using AttachAndDetach marks for cases where the attach is a new insert.
As we dicussed I can't make that work yet because we need to keep the insert effect (so we can compare its ID to the the ID of the cell).
That said, this change is still good. I've just pushed a pair of tests that fail if getInputCellId is not used.